lib/main.js

```diff
@@ -353,16 +352,17 @@ export default {
           }
 
           const msg = {
-            type: message.type,
-            filePath,
-            range,
+            severity: message.type.toLowerCase(),
```

Comment:
severity can _only_ be one of `error | warning | info`, unless PHPCS only uses those values, you need to translate whatever it gives into one of those values.